source/blog/2017-09-11-rspec-3-book-released.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Effective Testing With RSpec 3 has been released!
+author: Myron Marston
+---
+
+As we [announced back in February](/blog/2017/02/now-in-beta-effective-testing-with-rspec-3/),
+Ian Dees and I have written a new book for the Pragmatic Bookshelf
+about RSpec, entitled _Effective Testing with RSpec 3: Build Ruby Apps with Confidence_.
+We're happy to announce that the book is out of beta and is now shipping from
+[pragprog.com](https://pragprog.com/book/rspec3/effective-testing-with-rspec-3) and from
+[amazon.com](https://www.amazon.com/Effective-Testing-RSpec-Build-Confidence/dp/1680501984/).
+
+We put a ton of work over the last three years into this book, distilling our
+combined 20 years of testing experience into a book that explains how to do
+TDD/BDD well using RSpec. We believe that anyone who wants to learn RSpec or
+that regularly uses RSpec will benefit from the book.
+
+If you want to learn more about the book, check out this video Ian put together about it:
+
+<iframe class="centered" width="560" height="315" src="https://www.youtube.com/embed/-EbelD3CxAI" frameborder="0" allowfullscreen></iframe>
+
+You can also
+[check](https://media.pragprog.com/titles/rspec3/intro.pdf)
+[out](https://media.pragprog.com/titles/rspec3/writing.pdf)
+[multiple](https://media.pragprog.com/titles/rspec3/testing.pdf)
+[extracts](https://media.pragprog.com/titles/rspec3/origins.pdf)
+from the book to get an idea of what it's like.
```

Comment:
@undees I'm sure there's more we could say here but I'm not sure what.  Also, I'm tired and need to go to bed :).

Reply:
Hmm, one idea is some of the praise quotes...or do we want to tone down the self-promotion here?

Reply:
Yah, I like the idea of picking our favorite praise quote, and maybe introducing it with something like, "We're excited about the book because we finally finished it after pouring our souls into it. But we're not the only ones who are excited. Fakey McFakerson, CEO of Fakerson Industries, Ltd., said that it was, '....'"